shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -49,15 +49,18 @@ static void IOSPlatformThreadConfigSetter(const fml::Thread::ThreadConfig& confi
   // set thread priority
   switch (config.priority) {
     case fml::Thread::ThreadPriority::BACKGROUND: {
+      pthread_set_qos_class_self_np(QOS_CLASS_BACKGROUND, 0);
       [[NSThread currentThread] setThreadPriority:0];
       break;
     }
     case fml::Thread::ThreadPriority::NORMAL: {
+      pthread_set_qos_class_self_np(QOS_CLASS_DEFAULT, 0);
       [[NSThread currentThread] setThreadPriority:0.5];
       break;
     }
     case fml::Thread::ThreadPriority::RASTER:
     case fml::Thread::ThreadPriority::DISPLAY: {
+      pthread_set_qos_class_self_np(QOS_CLASS_USER_INTERACTIVE, 0);
```

Comment:
I think even just an info log with the `errno` and `strerror` in case this call fails (non-zero per the header) would be useful.

Reply:
These are just hints. We should avoid adding logs that are inactionable.

Reply:
I'm worried about totally glossing over a failure case I've run into in the past where setting extremely high priorities was reserved for certain system processes. So while you elevate the priorities a bit, setting the highest priorities would fail silently. Components that needed the highest priorites would then be run on threads with default priorities. I think in this case, it is somewhat fine since there aren't multiple priorities higher than default.

You call on the logs.

Reply:
How would adding an info log help with this case?

Reply:
I run with info logs enabled when debugging locally. Maybe I'd spot it 🤷🏽

Reply:
I don't think its reasonable to expect you or anyone else to visually diff the info logs on startup.

Reply:
sgtm